### DISCUSSION TITLE
Issues to install ASReview on python

### DISCUSSION THREAD

**Discussion by cchiara1** [asreview/asreview#1631] [Q&A]
Dear community,

I am trying to install ASReview. However, when I follow the instructions given by ASReview (here:https://asreview.readthedocs.io/en/latest/installation.html), the message "invalid syntax" appears. Any advice to solve this issue?

Thank you in advance.

**Comment by Rensvandeschoot** [comment]
Thank you for reporting! Can you copy the text displayed in the command line interface?

**Comment by Rensvandeschoot** [comment]
Can it be you opened Python CLI.exe instead of the CMD.exe? Detailed instructions can be found in our free, open teaching materials: https://asreview.github.io/asreview-academy/ASReviewLAB.html
